crates/nu-command/tests/commands/parse.rs

```diff
@@ -197,4 +197,21 @@ mod regex {
 
         assert_eq!(actual.out, "table<letter: string, a: string> (stream)")
     }
+
+    #[test]
+    fn parse_does_not_truncate_list_streams() {
+        Playground::setup("parse_test_streaming_1", |dirs, _sandbox| {
+            let actual = nu!(
+                cwd: dirs.test(), pipeline(
+                r#"
+                    [a b c]
+                    | each {|x| $x}
+                    | parse --regex "[ac]"
```

Comment:
Shouldn't this be `| parse --regex "([ac])"`? I think it works either way but `[ac]` produces no usable output since there isn't a capture group.

Reply:
Yeah, it doesn't really matter for the test, but I'll add the parens since it makes the test slighty more understandable when looking at the output of that parse manually.